demo/actions.py

```diff
@@ -57,50 +194,13 @@ def run(self, dispatcher, tracker, domain):
         gdrive = GDriveService()
         try:
             gdrive.store_data(sales_info)
-            return [SlotSet('data_stored', True)]
+            dispatcher.utter_template('utter_confirm_salesrequest', tracker)
```

Comment:
was there any occurrence of the slot `data_stored` in our stories?

Reply:
there was before, but before there weren't even any stories to handle if the value returned false. i took them all out (project-searched to make sure) and put the logic in here -- it removes one possible branching in the stories

Reply:
cool